doc/TOPP_tutorial/TOPP_scripting.doxygen

```diff
@@ -34,6 +34,269 @@
+
+        @image html TOPP_qualitycontrol.png
+        @image latex TOPP_qualitycontrol.png "" width=10cm
+
+
+        @section  TOPP_example_qualitycontrol_section3 Metrics
+        This Table shows what each of the included metrics does, what they need to be executed and what they add to
+        the data or what they store in their output.
+        <TABLE border=1>
+
+        <TR>
+            <TH colspan="2">
+                <b><br>Contaminants</b>
+                </TH>
+        </TR>
+        <TR>
+            <TD>
+                <b>Essential input data:</b> Fasta file (Contaminants database), PostFDR FeatureXML
+            </TD>
+            <TD>
+                <b> Output</b>
+            </TD>
+        </TR>
+        <TR>
+            <TD>
+                @b Brief: <br>
+                This metric checks whether the peptides are contaminants or not.<br>
+                @b Description: <br>
+                The Contaminants metric takes the contaminants database and digests the protein sequences with the
+                digestion enzyme that is given in the featureXML.
+                Afterwards it checks whether each of all peptide sequences of the featureXML (including the
+                unassigned peptide identifications) is registered in the contaminants database. <br>
+                To all peptides that are identified as a contaminant, the metric will add the metavalue
+                'is_contaminant  = 1' to the first hit of the peptide identification of all features as well
+                as to every unassigned peptideIdentification. <br>
+                Peptides that could not match any contaminant get an 'is_contaminant = 0' as a new metavalue.
+
+            </TD>
+            <TD>
+                <b> Changes in files:</b> <br>
+                This metric adds the new metavalue 'is_contaminant' into the first hit of each
+                peptideidentification and sets it to '1' if the peptide sequence is found in the database
+                and sets a '0' if not.<br>
+                <b> Other outputs:</b> <br>
+                For each FeatureMap the metric calculates the contaminant ratio of 1.) all peptides,
+                2.) all assigned peptides and 3.) all unassigned peptides as well as the intensity
+                ratio of all contaminants in the assigned peptides. Furthermore the number of empty
+                features and the number of all found features is also stored.
+            </TD>
+        </TR>
+
+        <TR>
+            <TH colspan="2">
+                <br><b>FragmentMassError</b>
+            </TH>
+        </TR>
+        <TR>
+             <TD>
+                  <b>Essential input data:</b> PostFDR FeatureXML, raw mzML file
+             </TD>
+             <TD>
+                  <b>Output</b>
+             </TD>
+        </TR>
+        <TR>
+             <TD>
+                  @b Brief: <br>
+                  This metric computes the FragmentMassError in ppm.<br>
```

Comment:
´... computes a list of fragment mass errors for each annotated MS2 spectrum in ppm and Da´